src/foundation_models/senpamae_wrapper.py

```diff
@@ -15,40 +15,43 @@
 
 
 class SenPaMAEClassification(LightningTask):
-    url = "TBD" #waiting on weights
+    url = "https://drive.google.com/file/d/16IoG47yzdyUnPqUgaV8ofeja5RgQjlAz"
```

Comment:
Should also be possible to do an automatic download for senpae with torchvision `download_url`